### PR TITLE
fix: replace edit with edit icon

### LINKED ISSUE #174: Replace edit with edit icon
We shoud replace the edit text for editing Ward and Organisation with a edit icon

### PR DESCRIPTION
**Pull request by rajarshisamaddar**
This pull request replace the edit text on my organization and the ward page with an edit icon.

Closes #174 Replace edit with an edit icon.

Testing instructions:
1. Install all the required packages from the package.json file.
2. Go to my organization page and notice the edit text is replaced with an edit icon.
3. Go to the ward page and notice the edit text is replaced with an edit icon.

Additional notes:
- Added one extra package that is react-icons that must be installed before use.

### CODE REVIEW

**Comment by florian-sabonchi** [comment]
Hello thank you for your pull request I think what concerns that is our contact person  @JannikWibker I have requested a review from him

**Comment by florian-sabonchi** [comment]
If there is interest we also have a Slack for communication @rajarshisamaddar

**Review by JannikWibker — COMMENTED** [review]
don't use font awesome, use proper icons such as https://lucide.dev instead

**Comment by rajarshisamaddar** [comment]
> don't use font awesome, use proper icons such as https://lucide.dev instead

@JannikWibker ok I'll fix this

**Comment by rajarshisamaddar** [comment]
@florian-sabonchi @JannikWibker review my changes using https://lucide.dev/

File: tasks/icons/Edit.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { Edit } from "lucide-react";
+
+const EditIcon = () => {
+  return <Edit color="black" size={24} />;
```
Comment (florian-sabonchi) [inline tasks/icons/Edit.tsx:4]: It looks quite good, but I think there is no need for an extra component for the icon.

**Comment by rajarshisamaddar** [comment]
> Looks good to me the last and final step is to solve merge conflicts. Whats your opinion @JannikWibker

@florian-sabonchi can you please tell me what is this merge conflict about I'm not getting it

**Comment by florian-sabonchi** [comment]
> > Looks good to me the last and final step is to solve merge conflicts. Whats your opinion @JannikWibker
> 
> @florian-sabonchi can you please tell me what is this merge conflict about I'm not getting it

The branch is several commits behind the main branch. Thereby there are changes that overlap which leads to problems. But i can fix that

**Comment by rajarshisamaddar** [comment]
> > > Looks good to me the last and final step is to solve merge conflicts. Whats your opinion @JannikWibker
> > 
> > 
> > @florian-sabonchi can you please tell me what is this merge conflict about I'm not getting it
> 
> The branch is several commits behind the main branch. Thereby there are changes that overlap which leads to problems. But i can fix that. But i wait for Jannik then i will solve and merge

OK

File: tasks/package.json [hunk 2]
```diff
@@ -31,7 +31,6 @@
     "react-device-detect": "^2.2.3",
     "react-dom": "^18.2.0",
     "lucide-react": "^0.145.0",
-    "simplebar-react": "^3.2.3",
```
Comment (use-to) [inline tasks/package.json]: I guess you never handled a merge conflict before.
That's not a problem.

Our current `main` branch needs this package in order to be operational.
As you added another package `lucide-react` to this project, your branch (`rajarshisamaddar:issue/174-replace-edit-with-edit-icon`) is not compatible with `helpwave:main` anymore cause another pull request demanding on `simplebar-react` got merged earlier.

To resolve this issue you normally just try to merge the `main` into this branch but you have to deal with conflicts manually. So just add both packages manually and do not delete the package introduced by these latest change in `main`.

Really good work, so far! 🚀 
We appreciate your cooperation very much and would be happy if you continue to help us develop our open source product (mozilla licensed). We have some minor bugs and issues with the label `low-prio`.

**Review by use-to — CHANGES_REQUESTED** [review]

**Comment by use-to** [comment]
@rajarshisamaddar If you want to work one more issues just tag @florian-sabonchi and he will take care about it as he's part of our core maintainer team.
We you want to receive more information and discuss about our vision feel free to join our Slack channel. Just write an mail to contact@helpwave.de and let us know!

File: tasks/package.json (hunk 2 above)
Comment (rajarshisamaddar) [reply tasks/package.json]: Yeah, this is the first time with the merge conflict. I'll try to solve this conflict. @use-to I would like to be a part of this project.

**Comment by rajarshisamaddar** [comment]
@florian-sabonchi I would like to work on more issues like this.

**Comment by florian-sabonchi** [comment]
image

**Review by JannikWibker — APPROVED** [review]

**Review by use-to — APPROVED** [review]

**Comment by use-to** [comment]
Approval overkill :D

> > Hello @rajarshisamaddar i fixed the issues with the pipeline and the merge conflicts now it can be approved and merged
> 
> Hi, @florian-sabonchi as you have fixed the pipeline issues what is my work on this now?

Just go ahead and take a look at our issues. You're job for this part is done.

**Pull request merged**